src/payment-agents/PaymentAgentsList.js

```diff
@@ -0,0 +1,26 @@
+import React, { PropTypes } from 'react';
+import PaymentAgentsItem from './PaymentAgentsItem';
+
+export default class PaymentAgentsList extends React.Component {
+    static propTypes = {
+        paymentAgents: PropTypes.object.isRequired,
+    };
+
+    render() {
+        const { paymentAgents } = this.props;
+        return (
+            <table>
+                <thead>
+                    <tr>
+                        <th>Payment Agent</th>
+                        <th>Commissions</th>
+                        <th>Banks</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    { paymentAgents.map(pa => <PaymentAgentsItem key={pa.name} paymentAgent={pa.toJS()}/>) }
```

Comment:
You better call paymentAgents.toJS() instead of on each item.
But we should do that conversion via selectors (or reselect).